libs/input-mapping/src/Helper/BuildQueryFromConfigurationHelper.php

```diff
@@ -10,14 +10,16 @@ class BuildQueryFromConfigurationHelper
 {
     public static function buildQuery(array $configuration): string
     {
-        if (isset($configuration['query']) && isset($configuration['source']['tags'])) {
+        $tagsPresent = isset($configuration['source']['tags']) && count($configuration['source']['tags']) > 0;
```

Comment:
Co to zjednodušit na `count($configuration['source']['tags'] ?? []) > 0`?

Reply:
Dobrý postřeh, upravil jsem, dík.